health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/UpdateChecklist.js

```
@@ -172,31 +227,31 @@ const UpdateChecklist = () => {
                 if (question.isActive) {
                     activeCounters[nestingKey] += 1;
                     // Build the counter string (e.g., "2.1" or "2.1.3")
                     parentCounter = parentCounter + '.' + activeCounters[nestingKey];
                 }
             }
-            
+
             codes[question.id] = code;
-    
+
             let moduleChecklist = "hcm-checklist";
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'moduleChecklist' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)